src/scripts.js

```diff
@@ -1,15 +1,156 @@
+const userBookingDashboard = document.getElementById('bookingsDashboard');
+const message = document.getElementById('message')
 
+const bookingsDashboard = document.getElementById('bookingsDashboard');
 
-console.log('This is the JavaScript entry file - your code begins here.');
+const currentBookings = document.getElementById('currentBookings');
+const upcomingBookings = document.getElementById('.upcomingBookings');
+const pastBookings = document.getElementById('.pastBookings');
 
-//♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢Event Listeners♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢//
-// window.onload()
+const startDate = document.querySelector('.date-picker');
+const bookRoomForm = document.getElementById('bookRoomForm');
 
-//♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢Event Handlers♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢♢//
+const userView = document.querySelector('.user-view');
+const loginPage = document.querySelector('.login-page');
+const loginSubmitButton = document.querySelector('.submit-button');
+// const usernameInput = document.querySelector('.username');
+// const passwordInput = document.querySelector('.password');
+//------------------Global Variables -----------------
+let customersData;
+let bookingsData;
+let roomsData;
+let currentUser;
+let overlookHotel;
+let getData;
+let postData;
+//------------------API CALLS -----------------
+const fetchData = () => {
+  Promise.all([customersAPI, roomsAPI, bookingsAPI, getRequest, postRequest])
+    .then(data => {
+      handleData(data)
+    });
+};
+
+
+//------------------Event Listeners------------------
+
+window.addEventListener('load', fetchData());
+
+//------------------Event Handlers-------------------
+const loadOverlook = (customersData, roomsData, bookingsData, getData, postData) => {
+  let i = Math.floor(Math.random() * customersData.length);
+  let customer = new Customer(customersData[i], bookingsData, roomsData);
+  loadCustomerDashboard(customer);
+  // let hotel = new Hotel(customersData, bookingsData, roomsData);
+  console.log(customer)
+  // console.log(new Date ())
+
+
+  // createEventListeners(customer, customersData, roomsData, bookingsData, postData, getData);
+};
+
+const handleData = (data) => {
+  customersData = data[0];
+  roomsData = data[1];
+  bookingsData = data[2];
+  getData = data[3];
+  postData = data[4];
+  loadOverlook(customersData, roomsData, bookingsData, getData, postData);
+};
+
+// const updateBookings = (e) => {
+//     e.preventDefault();
+//     getAllData(chosenUserID);
+// }
+//
+const loadCustomerDashboard = (customer) => {
+  customer.bookings.forEach((booking) => {
+     let today = new Date();
+     let bookingDate = new Date(booking.date);
+     if (bookingDate < today) {
+      //add this to my past booking
+      //PAST
+      currentBookings.innerHTML+= bookingPreview(booking)
+
+     } else if (bookingDate > today){
+       //UPCOMING
+
+
+    } else if (bookingDate > today) {
+      //CURRENT
+
+
+    }
+
+  })
+
+}
+
+// let current = customer.getCurrentBookings();
```

Comment:
Hope I can get rid all of this as I continue to refactor my customer dashboard! Leaving in for now